Tests/Reqnroll.GeneratorTests/UnitTestProvider/MsTestGeneratorProviderTests.cs

```diff
@@ -121,7 +121,7 @@ public void MsTestGeneratorProvider_ExampleSetSingleColumn_ShouldSetDescriptionW
             var converter = sampleTestGeneratorProvider.CreateUnitTestConverter();
 
             // ACT
-            var code = converter.GenerateUnitTestFixture(document, "TestClassName", "Target.Namespace");
+            var code = converter.GenerateUnitTestFixture(document, "TestClassName", "Target.Namespace", out var generationWarnings);
```

Comment:
Could we ignore the output with `out _` or better assert that no warnings happen?

Reply:
Added assertion for an empty enumerable.